vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputEventEmitter.h

```diff
@@ -53,6 +53,7 @@ class WindowsTextInputEventEmitter : public ViewEventEmitter {
   void onKeyPress(OnKeyPress value) const;
   void onContentSizeChange(OnContentSizeChange value) const;
   void onPressIn(GestureResponderEvent event) const override;
+  void onPressOut(GestureResponderEvent event) const;
```

Comment:
I noticed the lack of `override` on this method. I guess `onPressOut` isn't a virtual on `ViewEventEmitter`? Is it supposed to be?